spec/features/manage_proposals/manage_proposal_view_spec.rb

```diff
@@ -0,0 +1,120 @@
+describe 'Proposals listed on the Manage Proposals (MMT) page', js: true do
```

Comment:
remove js:true?

Reply:
The four files in features/manage_proposals require the js:true because it is either manipulating modals or the tab highlighting (features we are testing).